src/components/DrugChartSlider/DrugChartSlider.scss

```diff
@@ -0,0 +1,176 @@
+.dose-field-with-units {
+  display: flex;
+  flex-direction: row;
+  width: 55%;
+  justify-content: flex-start;
+  align-items: flex-end;
+  margin-right: 10px;
+
+  .dropdown {
+    width: 142.449px;
+  }
+}
+
+.duration-field-with-units {
+  display: flex;
+  flex-direction: row;
+  width: 55%;
+  justify-content: flex-start;
+  align-items: flex-end;
+  margin-right: 10px;
+
+  .dropdown {
+    width: 142.449px;
+  }
+}
```

Comment:
these two classes have same properties, any reason for having creating 2 diff classes?